### PR TITLE
bsunanda:Phase2-hgx65 New geometry for HGCAL

### PR DESCRIPTION
**Pull request by bsunanda**
This gives provision for covering the holes in the HGCAL geometry. Only set to work for SIM only at this moment. Changes for Digi-Reco will come later

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/data/v8/hgcalCons.xml [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+    <PartSelector path="//HGCalHECell.*"/>
+    <Parameter name="Volume" value="HGCalHECell" eval="false"/>
+    <Parameter name="CellSize"      value="[hgcalwafer:CellWF]"/>
+    <Parameter name="CellSize"      value="[hgcalwafer:CellWC]"/>
+  </SpecPar>
+  <SpecPar name="HGCalEEReco">
+    <PartSelector path="//HGCalEESensitive.*"/>
+    <Parameter name="Volume" value="HGCalEESensitive" eval="false"/>
+    <Parameter name="Slope"      value="[etaMax:slope]"/>
+    <Parameter name="GroupingZFine"  value="1"/>
```
Comment (ianna) [inline Geometry/HGCalCommonData/data/v8/hgcalCons.xml]: @bsunanda - why Vector is not used?

**Comment by ianna** [comment]
@bsunanda - just a general note. Using scram build with debug features is better then modifying the code every time you need to print debug messages:
```
scram b USER_CXXFLAGS=-DEDM_ML_DEBUG
```
also using Vector in xml rather then named parameter is safer.

**Comment by ianna** [comment]
+1

**Comment by bsunanda** [comment]
@ianna I shall try to take care of this later

**Comment by bsunanda** [comment]
@civanch Could you approve this so that we can go to HGCal TDR geometry sooner?

**Comment by civanch** [comment]
+1
I would agree that the debug method using ifdef is not very elegant but can stay at the initial phase of development

**Comment by davidlange6** [comment]
+1

**Pull request merged**